client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -88,9 +88,7 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.nio.charset.Charset;
-import java.util.List;
-import java.util.Locale;
-import java.util.StringJoiner;
+import java.util.*;
```

Comment:
`.*` imports aren't allowed in Elasticsearch code, which is why the test run failed - I suspect this is a change your IDE made automatically.

You can find guidance on how to configure your IDE [in our Contributing guide](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md) in the "Java Language Formatting Guidelines" section.